internal/pkg/csumio/csumwriter.go

```diff
@@ -0,0 +1,89 @@
+package csumio
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/cespare/xxhash/v2"
+)
+
+const ChecksumLen = 8
+
+type ChecksumWriter struct {
+	dataFile     io.Writer
+	checksumFile io.Writer
```

Comment:
The following comment isn't reflected.

https://github.com/cybozu-go/fin/pull/162/files#r2517908560